doc_src/en/HowTo_UseTeamProject.xml

```diff
@@ -2,7 +2,7 @@
 <!DOCTYPE section PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
 "../../../docbook-xml-4.5/docbookx.dtd">
 <section id="how.to.use.team.project">
-  <title id="how.to.use.team.project.title">Use a Team Project</title>
+  <title id="how.to.use.team.project.title">Use a team project</title>
```

Comment:
I would have thought the case for titles would remain the same?

Reply:
Lack of explanation on my part.

We've been using title case for chapter headings, and sentence case for everything below that.

This one happens to be an inconsistency I noticed at the same time, and I added it before I forgot to fix it.